configs/ruby/Ruby.py

```diff
@@ -331,8 +331,8 @@ def send_evicts(options):
     # 2. The x86 mwait instruction is built on top of coherence invalidations
     # 3. The local exclusive monitor in ARM systems
 
-    if isinstance(
-        options.cpu_type, DerivO3CPU
-    ) or ObjectList.CPUList().get_isa(options.cpu_type) in [ISA.X86, ISA.ARM]:
+    if isinstance(options.cpu_type, DerivO3CPU) or ObjectList.cpu_list.get_isa(
```

Comment:
@abmerop just making sure you saw this, not sure why it didn't affect the GPU stuff too?

Reply:
I modified GPU_VIPER.py to basically do this instead of calling this function... Looks like this will fix the issue though. Admittedly I put that patch together as quickly as possible since release is imminent so it would be worth reverting that particular hunk at some point

Reply:
My thought process is I am hoping to switch to a VIPER stdlib board for v24.0 and therefore this won't be used after that